docs/visualizing-your-data/snowplow-web/v1/index.md

```diff
@@ -0,0 +1,338 @@
+---
+title: "Marketing Dashboard Templates (v1)"
+date: '2023-06-01'
+sidebar_position: 0
+---
+```mdx-code-block
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import ThemedImage from '@theme/ThemedImage';
+```
+
+## Introduction
+
+This guide is written to introduce you to the dashboard templates designed for visualising your derived tables produced by the **snowplow-web** dbt package. It will show you how to get started and what each dashboard represents.
+
```

Comment:
Link to the package docs. I'd also maybe callout here that these are meant to get people started, we don't cover how to customise or add new things etc?